src/providers/LocalVideoProvider/docs/LocalVideoProvider.stories.mdx

```diff
@@ -2,7 +2,13 @@
 
 # LocalVideoProvider
 
-The `LocalVideoProvider` provides the local video status, video tile id associated with the local user's video and a function to toggle their video.
+The `LocalVideoProvider` provides video tile id associated with the local user's video, the local video status, a function to set the value of `isVideoEnabled` and a function to toggle their video.
```

Comment:
Can we extend "the local video status" -> "`isVideoEnabled` specifying the local video status"?

"The `LocalVideoProvider` provides video tile id associated with the local user's video, `isVideoEnabled` specifying the local video status, a function to set the value of `isVideoEnabled` and a function to toggle user's local video."